plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/TransformationMatrix.cpp

```diff
@@ -1118,12 +1157,15 @@ void TransformationMatrix::initialiseToPrim(bool readFromPrim, Transform* transf
     }
   }
 
-  // if some animation keys are found on the transform ops, assume we have a read only viewer of the transform data.
+  // Reenable push to prim if it was enabled before.
+  // (Note - I don't think we want to trigger an update of anything here, only
+  // reenable push for future operations, hence not calling enablePushToPrim.)
+  //
+  if(pushOriginallyEnabled)
+    m_flags |= kPushToPrimEnabled;
+
   if(m_flags & kAnimationMask)
-  {
-    m_flags &= ~kPushToPrimEnabled;
     m_flags |= kReadAnimatedValues;
```

Comment:
In our problem case the user had animated the translations/rotations on some models in her scene and saved her Maya file.  When she reopened the Maya file later to continue working, she could select those models but could no longer translate them - she would move the manipulator, but that's all that would move, the model stayed where it was.  Commenting out the pushToPrim disabling took care of this.

Reply:
Ok - I can see how disabling pushToPrim might lead to the behavior you're describing, which definitely isn't desired.
However, I'm a little worried that NOT disabling it might also make the tweak not work as I assume it was intended to allow the maya user to apply a constant offset on top of the animation values coming from USD.  Personally, though, I've never made use of this - or even confirmed that this is, indeed, what it was supposed to do... and that it was working.
Perhaps we can get someone from AL to comment here?